internal/configs/virtualserver_test.go

```diff
@@ -1960,3 +1961,44 @@ func TestGenerateEndpointsForUpstream(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateSlowStartForPlusWithInCompatibleLBMethods(t *testing.T) {
+	name := "test-slowstart-with-incompatible-LBMethods"
+	upstream := conf_v1alpha1.Upstream{Service: name, Port: 80, SlowStart: "10s"}
+	expected := ""
+
+	for lbMethod, _ := range incompatibleLBMethodsForSlowStart {
```

Comment:
it is better to have an array of the input parameters like you have in the `TestGenerateSlowStartForPlus`. this will make the tests much more readable.